ganga/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -313,7 +313,11 @@ def master_submit(self, rjobs, subjobconfigs, masterjobconfig, keep_going=False,
             upperlimit = (i+1)*nPerProcess
             if upperlimit > len(rjobs) :
                 upperlimit = len(rjobs)
-            logger.info("Submitting subjobs %s to %s" % (i*nPerProcess, upperlimit-1))
```

Comment:
Can you add back in the part about nPerProcess. It is useful with the block submission of jobs that is now done to Dirac.